config/locales/views/annotations/en.yml

```diff
@@ -2,10 +2,12 @@ en:
   annotations:
     count: 'Number of times used: '
     default_deduction: 'This annotation will have no deduction. Admins can configure deductions in assignment settings.'
+    empty_uncategorized: 'Currently, no annotations without a category exist.'
     help_message: "Click on an annotation category to add annotations."
     last_edited_by: "Last Edited By:"
-    text: "Text"
     list_deductions: 'Deductions from annotations: '
+    non_categorized: 'Non-Categorized Annotations'
```

Comment:
Hmm, actually here I think it's better to re-use the same "One-Time Only" label that appears in the annotation creation modal.

As for the key name itself, `uncategorized` is better than `non_categorized`.